HacktoberfestProject.Web/Views/Home/Add.cshtml

```diff
@@ -8,50 +8,59 @@
 
 <div class="container">
 	<form asp-controller="Home" asp-action="Add" method="post">
-		<div class="form-group">
-			<label asp-for="PrUrl"></label>
-			<input asp-for="PrUrl" class="form-control" oninput="processURL()" />
-			<span asp-validation-for="PrUrl" class="text-danger"></span>
-		</div>
 		<div class="form-group">
 			<label asp-for="Owner"></label>
-			<input asp-for="Owner" class="form-control" />
+			<input asp-for="Owner" 
+			       class="form-control"
+			       placeholder="Start typing the Github owner name..."
+			       autocomplete="off"/>
 			<span asp-validation-for="Owner" class="text-danger"></span>
 		</div>
+		<div class="form-group">
+			<label asp-for="PrUrl"></label>
+			<input asp-for="PrUrl" class="form-control" oninput="processURL()"/>
+			<span asp-validation-for="PrUrl" class="text-danger"></span>
+		</div>
 		<div class="form-group">
 			<label asp-for="Repository"></label>
-			<input asp-for="Repository" class="form-control" />
+			<input asp-for="Repository" class="form-control"/>
 			<span asp-validation-for="Repository" class="text-danger"></span>
 		</div>
 		<div class="form-group">
 			<label asp-for="PrNumber"></label>
-			<input asp-for="PrNumber" class="form-control" />
+			<input asp-for="PrNumber" class="form-control"/>
 			<span asp-validation-for="PrNumber" class="text-danger"></span>
 		</div>
 		<button type="submit" class="btn btn-primary">Add</button>
 		<button type="button"
-		class="btn btn-secondary"
-		onclick="window.location.href='/'">
+		        class="btn btn-secondary"
+		        onclick="window.location.href = '/'">
 			Cancel
 		</button>
 	</form>
 </div>
 
-<script type="text/javascript">function processURL() {
```

Comment:
If we're removing the `processUrl` function, we should also remove calling it from the input for `PrUrl`.
Was removing this intentional?

Reply:
Yes and good catch missed that

Reply:
See above, reverted decision.